src/main/java/seedu/duke/command/WatchlistCommand.java

```diff
@@ -91,12 +91,12 @@ private String selectWatchlist(Workspace activeWorkspace) throws AniException {
         Watchlist selectedWatchlist = watchlistList.get(selectIndex);
         Watchlist activeWatchlist = activeWorkspace.getActiveWatchlist();
         if (selectedWatchlist.equals(activeWatchlist)) {
-            LOGGER.log(Level.INFO, "Selected watchlist is currently the active watchlist.");
+            LOGGER.log(Level.INFO, "Select failed because the currently active watchlist is the selected.");
```

Comment:
Perhaps the phrasing for this message could be simpler

Reply:
Ok, I have removed some redundant words in it.